lib/src/presentation/bloc/card_bloc.dart

```diff
@@ -15,7 +17,6 @@ class CardBloc extends ICardBloc {
   int currentPage = kCardBlocStartingCurrentPage;
   late List<HearthstoneCard> allCards = [];
   int numberOfPagesCardList = kDefaultNumberOfPagesCardList;
-
   Map<int, List<HearthstoneCard>> page = {};
 
   CardBloc({required this.repositoryCard});
```

Comment:
Implement a usecase instead of the repository, and the usecase will do the repository call, to keep the correct Clean implementation

Reply:
Done.